tests/L0/run_transformer/test_fused_softmax.py

```diff
@@ -216,5 +224,69 @@ def test_autocast_fused_upper_triangle_mask_softmax(self):
                 actual.backward(g0)
                 expected.backward(g1)
 
+
+class TestGenericFusedSoftmaxKernel(common_utils.TestCase):
+
+    def _setup_qk(self):
+        qlen = [1, 2, 1234, 2322, 2348]
+        klen = [1, 2, 3, 4, 5, 8, 10, 11, 13, 128, 256, 1200, 1234, 2048, 3123, 4096, 4128, 7234, 8192, 10232]
```

Comment:
Is `10232` necessary here? Looks like it OOMs on GPUs even with 48GiB memory.

Reply:
It works on a 80G A100. 
It is not necessary though. Just to test some case with super long sequence length. I can remove it.

Reply:
Sure, it should also be possible to separate the very large cases and gate them based on available device memory e.g.,
https://github.com/pytorch/pytorch/blob/f50a248a5eacb9a9aa475a9e610486aea136e4f5/torch/testing/_internal/common_device_type.py#L914

Reply:
Thanks for sharing.  I think current test cases are enough to cover this new function.